seonshine_frontend/src/components/organims/userProfileModal/index.tsx

```diff
@@ -63,22 +64,35 @@ const UserProfileModal: React.FC<UserProfileModalProps> = ({ user, isOpen, onClo
   };
 
   const handleImageChange = async (event: React.ChangeEvent<HTMLInputElement>) => {
+    setUploadError(null);
     if (event.target.files && event.target.files.length > 0) {
       const file = event.target.files[0];
+      const validExtensions = ['image/png', 'image/jpeg', 'image/jpg'];
+      const maxFileSize = 5 * 1024 * 1024;
```

Comment:
move to global constant. Using name `maxFileSizeInByte` and `imgValidExtensions`